felles/src/main/kotlin/no/nav/helsearbeidsgiver/felles/EventTypes.kt

```diff
@@ -42,5 +43,8 @@ enum class EventName {
     FORESPØRSEL_MOTTATT,
     SAK_OPPRETTET,
     INNTEKTSMELDING_REQUESTED,
-    OPPGAVE_OPPRETTET;
+    OPPGAVE_OPPRETTET,
+    TRENGER_REQUESTED,
+    INNTEKT_REQUESTED,
+    EVENT_FOR_TEST
```

Comment:
Er ikke så fan av at Testdata "lekker" ut i applikasjon - går det an å slette BEHOV_FOR_TEST og EVENT_FOR_TEST, og heller benytte ekte verdier i testene?

Reply:
It has already leaked in the moment we have created enum in felles med alle mulige behov verdier istedenfor å ha behovene definert i modulene de er løst. Jeg er generelt ennig med deg, men akkuratt nå denne kommentar høres som å prøve å åpne en paraply når man er allerede våt :)

Reply:
Nei, det vil vel være mulig å kjøre testene med ekte behov og eventTypes, isteden for å bruke BEHOV_FOR_TEST og EVENT_FOR_TEST?

Reply:
Aha, nå skjønner jeg. Du vil at jeg setter en eksempel ekte event istedenfor dummy.